foursight_core/react/ui/index.html

```diff
@@ -0,0 +1 @@
+<link rel="shortcut icon" id="favicon" type="image/x-icon" href="https://cgap-dbmi.hms.harvard.edu/favicon.ico"/><script crossorigin src="https://unpkg.com/react@18.2.0/umd/react.development.js"></script><script crossorigin src="https://unpkg.com/react-dom@18.2.0/umd/react-dom.development.js"></script><script crossorigin src="https://unpkg.com/react-router@6.3.0/umd/react-router.development.js"></script><script crossorigin src="https://unpkg.com/react-router-dom@6.3.0/umd/react-router-dom.development.js"></script><script crossorigin src="https://unpkg.com/babel-standalone@6.26.0/babel.js"></script><!doctype html><html lang="en"><head><meta charset="utf-8"/><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/><link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous"/><link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous"/><meta name="viewport" content="width=device-width,initial-scale=1"/><meta name="theme-color" content="#000000"/><meta name="description" content="Web site created using create-react-app"/><link rel="manifest" href="/api/react/manifest.json"/><title>Foursight</title><script defer="defer" src="/api/react/static/js/main.js"></script><link href="/api/react/static/css/main.css" rel="stylesheet"></head><body><noscript>You need to enable JavaScript to run this app.</noscript><div id="root"></div></body></html>
```

Comment:
Is this file manually created or the result of a program? If it's human-maintained, I don't need to see lots of indentation but I wouldn't mind seeing newlines like:
```
<link rel="shortcut icon" id="favicon" type="image/x-icon" href="https://cgap-dbmi.hms.harvard.edu/favicon.ico"/>
<script crossorigin src="https://unpkg.com/react@18.2.0/umd/react.development.js"></script>
<script crossorigin src="https://unpkg.com/react-dom@18.2.0/umd/react-dom.development.js"></script>
...etc.
```
and at least a newline at the bottom. But without some line breaking, some editors not operating in wrap mode make this very hard to look at.